### PR TITLE
Update logs to include message data

### PR DESCRIPTION
**Pull request by teghnet**
This will make logs more precise

### CODE REVIEW

File: pkg/transport/p2p/oracle.go [hunk 1]
```diff
@@ -41,17 +41,27 @@ func oracle(feeders []ethereum.Address, signer ethereum.Signer, logger log.Logge
 			}
 			// Check is a message signature is valid and extract author's address:
 			priceFrom, err := priceMsg.Price.From(signer)
+			wat := priceMsg.Price.Wat
+			age := priceMsg.Price.Age
+			val := priceMsg.Price.Val
```
Comment (MDobak) [inline pkg/transport/p2p/oracle.go:46]:
```suggestion
			wat := priceMsg.Price.Wat
			age := priceMsg.Price.Age.UTC().Format(time.RFC3339)
			val := priceMsg.Price.Val.String()
```

File: pkg/transport/p2p/oracle.go [hunk 2]
```diff
@@ -41,17 +41,27 @@ func oracle(feeders []ethereum.Address, signer ethereum.Signer, logger log.Logge
...
+					WithField("age", age).
+					WithField("val", val).
 					Warn("The price message was rejected, invalid signature")
 				return pubsub.ValidationReject
 			}
 			// The libp2p message should be created by the same person who signs the price message:
 			if ethkey.AddressToPeerID(*priceFrom) != psMsg.GetFrom() {
 				logger.
 					WithField("peerID", psMsg.GetFrom().String()).
+					WithField("feedID", priceFrom.String()).
```
Comment (MDobak) [inline pkg/transport/p2p/oracle.go:61]:
```suggestion
					WithField("from", priceFrom.String()).
```

File: pkg/transport/p2p/oracle.go [hunk 3]
```diff
@@ -66,15 +76,21 @@ func oracle(feeders []ethereum.Address, signer ethereum.Signer, logger log.Logge
 			if !feedAllowed {
 				logger.
 					WithField("peerID", psMsg.GetFrom().String()).
-					WithField("feed", priceFrom.String()).
+					WithField("feedID", priceFrom.String()).
```
Comment (MDobak) [inline pkg/transport/p2p/oracle.go:79]:
```suggestion
					WithField("from", priceFrom.String()).
```

File: pkg/transport/p2p/oracle.go [hunk 4]
```diff
@@ -66,15 +76,21 @@ func oracle(feeders []ethereum.Address, signer ethereum.Signer, logger log.Logge
...
+					WithField("val", val).
 					Warn("The price message was ignored, the feeder is not allowed to send price messages")
 				return pubsub.ValidationIgnore
 			}
 			// Check when message was created, ignore if older than 5 min, reject if older than 10 min:
 			if time.Since(priceMsg.Price.Age) > 5*time.Minute {
 				logger.
 					WithField("peerID", psMsg.GetFrom().String()).
-					WithField("feed", priceFrom.String()).
+					WithField("feedID", priceFrom.String()).
```
Comment (MDobak) [inline pkg/transport/p2p/oracle.go:90]:
```suggestion
					WithField("from", priceFrom.String()).
```

**Review by MDobak — CHANGES_REQUESTED** [review]
Few changes to make field names consistent

File: config.json [hunk 5]
```diff
@@ -365,6 +365,20 @@
...
+          [{ "origin": "bitstamp", "pair": "MKR/USD" }],
+          [{ "origin": "coinbasepro", "pair": "MKR/USD" }],
+          [{ "origin": "ftx", "pair": "MKR/USD" }],
+          [{ "origin": "gemini", "pair": "MKR/USD" }],
+          [{ "origin": "kraken", "pair": "MKR/USD" }]
+        ],
+        "params": {
+          "minimumSuccessfulSources": 4
+        }
+      },
```
Comment (MDobak) [inline config.json:381]: ?

Comment (teghnet) [reply config.json:381]: I wanted to have it in gofer. It is useful and independent of prod.

**Review by MDobak — APPROVED** [review]

**Review by MDobak — APPROVED** [review]

**Pull request merged**